test/e2e_node/dynamic_kubelet_config_test.go

```diff
@@ -70,7 +70,7 @@ type nodeConfigTestCase struct {
 }
 
 // This test is marked [Disruptive] because the Kubelet restarts several times during this test.
-var _ = SIGDescribe("[Feature:DynamicKubeletConfig][NodeFeature:DynamicKubeletConfig][Serial][Disruptive]", func() {
+var _ = SIGDescribe("[Feature:DynamicKubeletConfig][NodeFeature:DynamicKubeletConfig][Deprecated][Disruptive]", func() {
```

Comment:
I want to remove `Feature:DynamicKubeletConfig` from the Serial lane as we discussed during the sig-node-ci meeting. It anyway should be removed under 1.23, so I think we can just remove it because we are already during the 1.23 release cycle.

Reply:
I think it is better to change the test-infra definition of the tab and exclude the test there than mix up the labels. What do you mean saying that "It should be removed in 1.23"? What is "it"?

Reply:
DynamicConfiguration feature, at least it what I can see
```
Feature DynamicKubeletConfig is deprecated in 1.22 and will not move to GA. It is planned to be removed from Kubernetes in the version 1.23. Please use alternative ways to update kubelet configuration
```

Reply:
The test-infra PR https://github.com/kubernetes/test-infra/pull/24234